recipe/meta.yaml

```diff
@@ -1,37 +1,38 @@
-{% set version = "0.20.9" %}
+{% set version = "0.20.10" %}
 
 package:
   name: pyjanitor
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyjanitor/pyjanitor-{{ version }}.tar.gz
-  sha256: 242bae9a2ca5d3e81648b9f8e3573dbe5e9f80a07cbb9f4e2b3fe59cd6e3b528
+  sha256: cde75bd4b67baedfdcf2c354a151bceecefb5f3b6501990c8777a5575c3a3f35
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - python >=3.6
     - pip
   run:
     - python >=3.6
-    - pandas
+    - natsort
     - numpy
-    - scikit-learn
+    - pandas
     - pandas-flavor
-    - natsort
     - requests
+    - scikit-learn
+    - xarray
     - xlrd
 
 test:
-  requires:
-    - pip
-  commands:
-    - pip check
+  # requires:
+  #   - pip
+  # commands:
+  #   - pip check
```

Comment:
@ericmjl I had to remove this test b/c of https://github.com/ericmjl/pyjanitor/issues/760

Let me know if we all of those extras deps are needed here.

Reply:
Merging this b/c it seems we have all the minimum requirements in the recipe, We cannot run `pip check` due to the over-specification of dependencies upstream but that is being worked out and will be solved soon.